lib/search_config.rb

```diff
@@ -131,6 +157,14 @@ def combined_index_schema
     )
   end
 
+  def meta_assign(method, cluster, &block)
```

Comment:
This is pretty clever, but I'm not sure of a more obvious way to do it.

Reply:
Yeah I asked around to see if anyone has ideas on a nicer way to do this (see our internal ruby channel 😄). If you have an idea I'd be interested!